addOns/graphql/src/test/java/org/zaproxy/addon/graphql/VariantGraphQIUnitTest.java

```diff
@@ -127,6 +129,29 @@ void shouldLogWarnOnInvalidJson() throws HttpMalformedHeaderException {
         assertThat(logMessages.size(), is(1));
     }
 
+    @Test
+    void shouldSetGraphqlQueryParametersCorrectlyOnTheUrl()
+            throws HttpMalformedHeaderException, URIException {
+        // Given
+        HttpRequestHeader httpReqHeader = new HttpRequestHeader();
+        httpReqHeader.setMessage(
+                "GET /graphql/?query=%7BsqlInjection(expression:%20%221%22)%7D HTTP/1.1");
```

Comment:
Add other parameter? Would be good to test the loop.

Reply:
Yes nice idea, added one before and one after.